storey/table.py

```diff
@@ -44,6 +46,20 @@ def __init__(self, table_path: str, storage: Driver, partitioned_by_key: bool =
     def __str__(self):
         return f'{self._container}/{self._table_path}'
 
+    def _get_lock(self, key):
+        key = get_hashed_key(key)
+        cache_element = self._attrs_cache.get(key)
+        if cache_element is None:
+            cache_element = _CacheElement({}, None)
+            self._attrs_cache[key] = cache_element
+        cache_element.lock = Lock()
+        return cache_element.lock
+
+    def _get_schema_lock(self):
+        if self._schema_lock is None:
```

Comment:
why not create the lock in ctor instead checking this if each time

Reply:
That wouldn't work, because it needs the event loop.